addons/uh60_weapons/config/cfgMagazines.hpp

```diff
@@ -28,14 +28,14 @@ class cfgMagazines {
     ammo = "VTX_Hellfire_AGM114N";
     pylonWeapon = "vtx_hellfire_launcher_N";
   };
-  class VTX_M230_Chaingun_L: VehicleMagazine {
+  class VTX_M230_Chaingun_L_HEDP: VehicleMagazine {
```

Comment:
keep the old weapon and magazine class names please, and have them use the new ammo. adding ap version sounds great.